c_src/main.c

```diff
@@ -591,9 +591,18 @@ static ERL_NIF_TERM erlfdb_network_set_option(ErlNifEnv *env, int argc,
 #if FDB_API_VERSION >= 630
     else if (IS_ATOM(argv[0], enable_run_loop_profiling)) {
         option = FDB_NET_OPTION_ENABLE_RUN_LOOP_PROFILING;
-    }
+    } else if (IS_ATOM(argv[0], client_threads_per_version)) {
+        option = FDB_NET_OPTION_CLIENT_THREADS_PER_VERSION;
```

Comment:
can you help me find a reference in the FDB source code that shows FDB does not create any external client threads when FDB_NET_OPTION_CLIENT_THREADS_PER_VERSION is set to a value of exactly 1?

Reply:
`fdbclient/MultiVersionTransaction.actor.cpp`:

https://github.com/apple/foundationdb/blob/10bf6bbfcaf8ddc5214cc806e94edc21eb18f4bc/fdbclient/MultiVersionTransaction.actor.cpp#L2875